### PR TITLE
Embed validation site application

### PR DESCRIPTION
**Pull request by guzman-raphael**
Depends on vathes/datajointio#69
Supersedes #20

image
image

### CODE REVIEW

File: apply.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Apply
+
+Thank you for your interest in becoming a validation site for DataJoint Elements!
```
Comment (dimitri-yatsenko) [inline apply.md:3]: To make more general, 
```suggestion
Thank you for your interest in engineering support to for adopting DataJoint Elements!
```

File: apply.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+To apply to participate as a validation site, please complete the form below and submit to us.
```
Comment (dimitri-yatsenko) [inline apply.md:5]:
```suggestion
Please complete the form below.
```

File: apply.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        <div style="float:left;">
+            <label for="lab">Lab:</label>
+            <br>
+            <input name="lab" type="text" id="lab" size="30">
+        </div>
+        <br style="clear:both;">
+    </section>
+    <section>
+        <div style="float:left;margin-right:20px;">
+            <label for="data_modalities">Describe Data Modalities (ephys, calcium imaging, etc.):</label>
```
Comment (dimitri-yatsenko) [inline apply.md:37]:
```suggestion
            <label for="data_modalities">Describe data modalities (ephys, calcium imaging, etc.):</label>
```

File: apply.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        <div style="float:left;margin-right:20px;">
+            <label for="current_workflow">Describe current workflow for data acquisition and analysis:</label>
+            <br>
+            <input name="current_workflow" type="text" id="current_workflow" size="67">
+        </div>
+        <br style="clear:both;">
+    </section>
+    <section>
+        <div style="float:left;margin-right:20px;">
+            <label for="mysql_setup">Do you prefer to setup a MySQL Server yourself or would you like to utilize cloud managed hosting?</label>
```
Comment (dimitri-yatsenko) [inline apply.md:53]:
```suggestion
            <label for="mysql_setup">Do you prefer to setup a database server yourself (on premises or cloud) or use DataJoint Neuro cloud managed hosting?</label>
```

File: apply.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+            <br>
+            <input name="mysql_setup" type="text" id="mysql_setup" size="67">
+        </div>
+        <br style="clear:both;">
+    </section>
+    <section>
+        <div style="float:left;margin-right:20px;">
+            <label for="start_date">When would you like to begin the tests?</label>
```
Comment (dimitri-yatsenko) [inline apply.md:61]:
```suggestion
            <label for="start_date">When would you like to schedule the first test?</label>
```

**Comment by guzman-raphael** [comment]
Thank you for the review @dimitri-yatsenko. Your suggestions have now been applied.

**Review by dimitri-yatsenko — APPROVED** [review]

**Pull request merged**